### PR TITLE
add flash message on committee page

### PR DESCRIPTION
**Pull request by mykiwi**
Render:
capture d ecran 2017-04-21 a 14 40 08

### CODE REVIEW

**Comment by tgalopin** [comment]
Can you add a message to say "Only on supervisor per committee allowed" ?

**Comment by mykiwi** [comment]
@tgalopin done

File: src/AppBundle/Exception/CommitteeMembershipException.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public static function createNotPromotableSupervisorPrivilegeException(UuidInter
     {
         return new self(
             $membershipUuid,
-            sprintf('Committee membership "%s" cannot be promoted to the supervisor privilege.', $membershipUuid),
+            sprintf('Committee membership "%s" cannot be promoted to the supervisor privilege. Only on supervisor per committee allowed.', $membershipUuid),
```
Comment (HeahDude) [inline src/AppBundle/Exception/CommitteeMembershipException.php]: on => one

Comment (mykiwi) [inline src/AppBundle/Exception/CommitteeMembershipException.php]: Good catch

**Pull request merged**